api/turing/config/config.go

```diff
@@ -469,6 +479,8 @@ func setDefaultValues(v *viper.Viper) {
 	v.SetDefault("RouterDefaults::FluentdConfig::Tag", "turing-result.log")
 	v.SetDefault("RouterDefaults::FluentdConfig::FlushIntervalSeconds", "90")
 	v.SetDefault("RouterDefaults::Experiment", map[string]interface{}{})
+	v.SetDefault("RouterDefaults::KafkaConfig::MaxMessageBytes", "1048588")
+	v.SetDefault("RouterDefaults::KafkaConfig::CompressionType", "none")
```

Comment:
Should this default value be `uncompressed` rather than `none`?

Ref: https://kafka.apache.org/documentation/#topicconfigs_compression.type

Reply:
Nope it is `none` because we are setting this for the producer config
https://kafka.apache.org/documentation/#producerconfigs_compression.type

Reply:
Ah saw the wrong doc, cool thanks!